lightning-invoice/src/payment.rs

```diff
@@ -2424,4 +2424,143 @@ mod tests {
 		nodes[0].node.process_pending_events(&invoice_payer);
 		assert!(expected_events.borrow().is_empty());
 	}
+
+	///	Helper function to forward HTLC update and RevokeAndAck messages
+	fn handle_message_send_events(sender:&Node, node_indices_by_id: &HashMap<PublicKey, usize>, nodes: &Vec<Node>) {
+		let mut to_notify : Vec<&Node>=Vec::new();
+		for msg in sender.node.get_and_clear_pending_msg_events() {
+			match msg {
+				MessageSendEvent::SendRevokeAndACK { node_id, msg } => {
+					println!("Sendrevoke {:?}", msg);
```

Comment:
Print statements should mainly be used for local testing purposes. You may leave them in testcode, however, it then would be nice for them to be full sentences and properly formatted.